docs/dashboards/frequently-asked-questions.md

```diff
@@ -127,6 +127,11 @@ Highcharts Dashboards supports the following browsers:
 | Opera    | 42.0+ (2016+) |
 | Edge     | 16.0+ (2017+) |
 
+* * *
+What versions of Highcharts are supported?
```

Comment:
Whenever referencing Highcharts from Dashboards docs, faqs etc. I think we should start calling it "Highcharts Core", since that's the actual product name.

Reply:
Fair point but with Dashboards we can use all product Core/Stock/Maps/Gantt and its modules.
Thus I'm not convinced if we should call it here like that.

Reply:
Ah, I see! OK, then I agree. But let's always use "Highcharts Core" when we mean core, moving forward.